.changeset/strong-pianos-attend.md

```diff
@@ -0,0 +1,7 @@
+---
+"@orbit-ui/transition-components": patch
+"@workleap/orbiter-ui": patch
+"@orbit-ui/transition-css": patch
+---
+
+Migrate from yarn and lerna to pnpm only
```

Comment:
unecessary, we don't have to release for that. I would remove this file entirely